.github/workflows/multiarch-build.yaml

```diff
@@ -0,0 +1,65 @@
+name: Multiarch build
+on:
+  pull_request:
```

Comment:
I'd prefer to only build in PR, not pushing.

Or if you want to push, it should be different tag, not latest

Reply:
I have changed the tag to `multiarch`.

Reply:
Okay we have to separate it into 3 workflows:
* pull_request: it shouldn't push image to registry, or if you want to pull it for testing purposes, it should be with unique tag `pr-<pr_number>`
* master branch: after merge should be built and pushed as latest tag
* release branches: should be pushed as particular release version tag (not latest)

Reply:
@MartinBasti I have updated tags for master branch and release event. Please review.